uds/transport_interface/can_transport_interface.py

```diff
@@ -302,11 +310,11 @@ def n_cr_timeout(self, value: TimeMillisecondsAlias):
 
     @property  # noqa
     @abstractmethod
-    def n_cr_measured(self) -> Optional[TimeMillisecondsAlias]:
+    def n_cr_measured(self) -> Optional[Tuple[TimeMillisecondsAlias, ...]]:
         """
-        Get the last measured value of N_Cr time parameter.
+        Get the last measured values (during the last message reception) of N_Cr time parameter.
```

Comment:
add reference to knowledge base N_Cr time explanation (same for other time parameters)